src/dialogs/more-info/controls/more-info-cover.js

```diff
@@ -23,11 +23,18 @@ import featureClassNames from '../../../common/entity/feature_class_names';
         max-height: 0px;
         overflow: hidden;
       }
+
+      .current_position > div,
+      .tilt > .name {
+        padding-top: 4px;
```

Comment:
This feels weird. We should try to have a single styling for all our sliders in more info.

Reply:
I'm open to suggestions. My reasoning was just that the slider use cases are already fairly customized and adding the padding to the top of the slider would look wired.

Reply:
We should use the same labeled slider everywhere and that should have the same look https://github.com/home-assistant/home-assistant-polymer/blob/master/src/components/ha-labeled-slider.js